start_cerberus.py

```diff
@@ -111,6 +112,24 @@ def main(cfg):
                              "to True and assuming that the nodes are ready")
                 watch_nodes_status = True
 
+            # Monitor cluster operators status
+            if watch_cluster_operators:
+                operators_status = runcommand.invoke("kubectl get co -o yaml")
+                status_yaml = yaml.load(operators_status, Loader=yaml.FullLoader)
+                watch_cluster_operators_status, failed_operators = \
+                    kubecli.monitor_cluster_operator(iteration, status_yaml)
+                logging.info("Iteration %s: Cluster Operator status: %s"
+                             % (iteration, watch_cluster_operators_status))
+            else:
+                logging.info("Cerberus is not monitoring cluster operators, "
+                             "so setting the status to True and "
+                             "assuming that the cluster operators are ready")
+                watch_cluster_operators_status = True
+            
+            if not watch_cluster_operators_status:
+                logging.info("Failed operators")
```

Comment:
Failed operators needs to logged after logging the failed nodes. Failed nodes and pods are logged after logging True/False status of all the namespaces in watch_namepsace.

Reply:
Hey Yashashree, I'm not sure I follow why the failed operators would need to be logged at a different time then when I get the failed operators. The status of the failed operators shouldn't change before or after the failed nodes, pods and namespaces. To me it just seems like it is splitting up the components of the code that are similar. What are your thoughts?

Reply:
Yeah, I thought it would be better to maintain the existing code style similar to how failed nodes and pods are reported after logging the True/False status of each namespace. This ensures that the report is clean. Or we could report failed nodes/operators/pods after reporting the status of the same. But the same style has to be maintained in all the cases. What are your thoughts @chaitanyaenr ?

Reply:
When we log all the failures related to the components being monitored including cluster operators, nodes, pods e.t.c in one place, it will make it easier for the user for see what happened during that iteration IMHO.

Reply:
Ok, that makes sense. I'll work on fixing that now.